measures/UpgradeCosts/measure.rb

```diff
@@ -117,14 +117,16 @@ def run(runner, user_arguments)
     end
 
     if not has_costs
-      register_value(runner, upgrade_cost_name, '')
-      runner.registerInfo("Registering (blank) for #{upgrade_cost_name}.")
+      register_value(runner, upgrade_cost_name, 0.0)
+      runner.registerInfo("Registering 0.0 for #{upgrade_cost_name}.")
       return true
     end
 
     # Obtain cost multiplier values and calculate upgrade costs
     upgrade_cost = 0.0
     option_cost_pairs.keys.each do |option_num|
+      next if option_cost_pairs[option_num].empty?
```

Comment:
Why was this necessary to add?

Reply:
The loop iterates through all 25 options, but we don't always have all 25 populated.